bci_tester/data.py

```diff
@@ -188,6 +188,9 @@ def create_BCI(
 GO_1_18_CONTAINER = create_BCI(
     build_tag="bci/golang:1.18", image_type="hybrid"
 )
+GO_1_19_CONTAINER = create_BCI(
+    build_tag="bci/golang:1.19", image_type="hybrid"
```

Comment:
this container is only available for 15.4

Reply:
Updated, please check again.